### DISCUSSION TITLE
Some components are not showing properly

### DISCUSSION THREAD

**Discussion by borjxrroyo** [saadeghi/daisyui#922] [Q&A]
Hello there,

I installed TailwindCSS and DaisyUI following the steps from the webpages. But some components are showing properly and others not.

Example working (dropdown with avatar online):

![image](https://user-images.githubusercontent.com/72812865/177053229-c5ee2f90-5568-4328-8560-cd1ee5d7f0f3.png)

Example not working (card):

![image](https://user-images.githubusercontent.com/72812865/177053255-fcce5aff-1805-45e3-ac39-e85b6f07b017.png)

![image](https://user-images.githubusercontent.com/72812865/177053266-a47a226b-2fd8-44ae-af6e-357280c8349d.png)

My config:

![image](https://user-images.githubusercontent.com/72812865/177053286-279bf454-647a-45c5-9b6f-6677dd45f86a.png)

![image](https://user-images.githubusercontent.com/72812865/177053295-83acf6f9-5748-4322-afbe-d796b184fbdb.png)

![image](https://user-images.githubusercontent.com/72812865/177053313-81d62d74-4fb9-4a4e-ba92-31b19d64fb41.png)

Thanks.

**Comment by saadeghi** [comment]
I couldn't reproduce this issue. can you please share an example repo?
